third_party/k8s.io/client-go/informers/externalversions/apps/interface.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2017 Jetstack Ltd.
```

Comment:
❓ This should be Copyright Kubernetes I guess.

Reply:
I'm not really sure to be honest - it's generated code, so I feel like it's not something that needs to be attributed to the k8s authors. If we were to change *this* to refer to the k8s authors, I feel like we should change *all* code generated by code-generators to be credited to the k8s authors.